src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
+        Machine[] machines = { new Truck(), new Bulldozer(), new Excavator() };
 
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
Similarly, the 'stopWork' method should also output an informative message specific to the machine type, such as 'Truck stopped its work.'